src/ui/mainwindow.h

```diff
@@ -134,17 +140,22 @@ private slots:
     QList<SearchState *> m_tabs;
 
     SearchState *m_searchState = nullptr;
-    Zeal::NetworkAccessManager *m_zealNetworkManager = nullptr;
 
-    Ui::MainWindow *ui = nullptr;
+    std::unique_ptr<Ui::MainWindow> ui;
     Zeal::Core::Application *m_application = nullptr;
     Zeal::Core::Settings *m_settings = nullptr;
-    Zeal::ListModel *m_zealListModel = nullptr;
-    Zeal::SettingsDialog *m_settingsDialog = nullptr;
+    std::unique_ptr<Zeal::NetworkAccessManager> m_zealNetworkManager;
```

Comment:
Same as with `QShortcut`, all these are `QObject`s with set parent, there's no need in any additional memory management.